cvat/apps/dataset_manager/task.py

```diff
@@ -431,13 +439,29 @@ def put(self, data):
             self._set_updated_date()
 
     def update(self, data):
+        data = self._validate_input_annotations(data)
+
         self._delete(data)
         self._create(data)
         handle_annotations_change(self.db_job, self.data, "update")
 
         if not self._data_is_empty(self.data):
             self._set_updated_date()
 
+    def _validate_input_annotations(self, data: Union[AnnotationIR, dict]) -> AnnotationIR:
+        if not isinstance(data, AnnotationIR):
+            data = AnnotationIR(self.db_job.segment.task.dimension, data)
+
+        db_data = self.db_job.segment.task.data
+
+        if data.tracks and db_data.validation.mode == models.ValidationMode.GT_POOL:
+            # Only tags and shapes can be used in tasks with GT pool
+            raise ValidationError("Tracks are not supported when task validation mode is {}".format(
+                models.ValidationMode.GT_POOL
+            ))
```

Comment:
So, as far as I understand we cant use tracks in tasks with HP. 
But when I try to add a track in UI I get this error:
![image](https://github.com/user-attachments/assets/988b1617-dd84-4224-9e2b-d2ae037b8e01)

Seems error message is wrong

Reply:
Thanks, fixed

Reply:
@zhiltsov-max  What is the nature of such restriction?
I understand that tracks with interpolation do not make sense in honeypot jobs, but is it the only reason?

Reply:
Frames are in the random order, tracks make no sense in such case.

Reply:
It makes sense if each frame of the track is a keyframe.

Reply:
Just do not understand the reason why we forced this restriction.
If a user does it, why not. It is the user's decision.